### DISCUSSION TITLE
Start button icon

### DISCUSSION THREAD

**Discussion by hosykynam0619ch2008-umego** [valinet/ExplorerPatcher#2766] [Q&A]
Hi there, I was trying to set the start button to Windows 10 icon as informed in the Wiki.
![image](https://github.com/valinet/ExplorerPatcher/assets/129523779/d65b7911-c7c3-4802-adb6-1ad18efbf4de)
However when I set it to Windows 10, it still displaying Windows 11 square icon
![image](https://github.com/valinet/ExplorerPatcher/assets/129523779/33fbf402-fd3e-41ce-a8ff-f5c17172b217)
Is there anyway to fix this bug, or please rollout an updates for this, I'm on version 22631.3007
![image](https://github.com/valinet/ExplorerPatcher/assets/129523779/7b442f5a-90e9-4300-945d-73903806f19b)
Thank you!

**Comment by hosykynam0619ch2008-umego** [comment]
Here's my setting btw if you need
![image](https://github.com/valinet/ExplorerPatcher/assets/129523779/9548ed70-348e-40c4-a50b-c9a99fb8c77d)

**Comment by pyrates999** [comment]
This may require the taskbar be set to windows 11 style.

**Reply by hosykynam0619ch2008-umego** [reply]
I want to change the start button icon into Windows 10, not 11, seems like 11 doesn't have the option to change the start button icon

**Comment by tallman69** [comment]
The option to change the Start button style is only available when the Taskbar style is set to Windows 10.  If the Start button style is changed, File Explorer must be restarted for the change to take effect.

**Reply by hosykynam0619ch2008-umego** [reply]
I already restarted the explorer after change the start button icon to windows 10

**Reply by pyrates999** [reply]
You must also set your taskbar to windows 10.

**Comment by hosykynam0619ch2008-umego** [comment]
Yes I did already!

Vào Th 4, 31 thg 1, 2024 lúc 7:58 CH pyrates999 ***@***.***>
đã viết:

**Comment by pyrates999** [comment]
How did you restart file explorer?

**Comment by hosykynam0619ch2008-umego** [comment]
I click on restart file explorer button on explorerpatcher (bottom-right)

Vào Th 4, 31 thg 1, 2024 lúc 8:12 CH pyrates999 ***@***.***>
đã viết:

**Comment by pyrates999** [comment]
open up task manager and kill explorer.exe and then run task manager again and run explorer.exe

**Comment by hosykynam0619ch2008-umego** [comment]
Did it, same thing…

Vào Th 4, 31 thg 1, 2024 lúc 9:07 CH pyrates999 ***@***.***>
đã viết:

**Comment by pyrates999** [comment]
Not sure then.  Please wait for a developer to respond.
